pkg/deploy/devconfig.go

```diff
@@ -157,9 +157,9 @@ func DevConfig(_env env.Core) (*Config, error) {
 				"qos.ppe.warm.ingest.monitor.core.windows.net",
 				"test1.diagnostics.monitoring.core.windows.net",
 			},
-			GatewayFeatures: []string{
-				"InsecureSkipVerifyDBTokenCertificate",
-			},
+			// GatewayFeatures: []string{
```

Comment:
Let's just rip this out completely if it's not going to be used.  No need to comment it out.

Reply:
Sure